### PR TITLE
CN-8588, CN-8589: Implement warning message

### PR DESCRIPTION
**Pull request by wp-ilija-vucetic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/HODLViewCode/HODLViewCode.js [hunk 1]
```diff
@@ -101,43 +102,78 @@ class HODLViewCode extends Component {
...
+                  <View style={style.blurImageAndIconWrapper}>
+                    <ThemedImage style={style.buttonIconHand}
+                                 lightSource={require("../../../../assets/images/hold-to-show.png")}
+                                 darkSource={require("../../../../assets/images/hold-to-show.png")}
+                                 unicornSource={require("../../../../assets/images/hold-to-show.png")}
+                    />
+                    <CelText link style={style.longPressText}>Long press to reveal</CelText>
+                  </View>
+                  <ImageBackground style={style.blurImage}
+                                   imageStyle={{ borderRadius: 6 }}
```
Comment (CrnogoracIvan) [inline app/components/screens/HODLViewCode/HODLViewCode.js:131]: Move borderRadius to style

File: app/components/screens/HODLViewCode/HODLViewCode.js [hunk 2]
```diff
@@ -101,43 +102,78 @@ class HODLViewCode extends Component {
...
-                  <CelText
-                    color={getColor(COLOR_KEYS.PRIMARY_BUTTON)}
-                    onPress={() =>
-                      this.setState({ showHodlCode: !this.state.showHodlCode })
-                    }
-                  >
-                    {showHodlCode ? "HIDE" : "SHOW"}
-                  </CelText>
+            <Card margin={"20 0 20 0"}>
+              <TouchableOpacity
```
Comment (CrnogoracIvan) [inline app/components/screens/HODLViewCode/HODLViewCode.js:115]: Maybe TouchableWithoutFeedback?

File: app/components/screens/HODLViewCode/HODLViewCode.js [hunk 3]
```diff
@@ -101,43 +102,78 @@ class HODLViewCode extends Component {
...
+                      {hodlCode}
+                    </CelText>
+                  </View>
                 </View>
-              </View>
+              </TouchableOpacity>
             </Card>
           )}
 
+          <InfoBox
```
Comment (CrnogoracIvan) [inline app/components/screens/HODLViewCode/HODLViewCode.js:146]: Move to renderInfoBox function

File: app/components/screens/HODLViewCode/HODLViewCode.js [hunk 4]
```diff
@@ -101,43 +102,78 @@ class HODLViewCode extends Component {
...
-                  >
-                    {showHodlCode ? "HIDE" : "SHOW"}
-                  </CelText>
+            <Card margin={"20 0 20 0"}>
+              <TouchableOpacity
+                activeOpacity={1}
+                onPressIn={() => this.setState({ showHodlCodeOverlay: false })}
+                onPressOut={() => this.setState({ showHodlCodeOverlay: true })}>
+
+                {this.state.showHodlCodeOverlay &&
```
Comment (CrnogoracIvan) [inline app/components/screens/HODLViewCode/HODLViewCode.js:120]: Maybe ternary here?

File: app/components/screens/TwoFactorSettings/TwoFactorSettings.js [hunk 5]
```diff
@@ -79,7 +77,40 @@ class TwoFactorSettings extends Component {
...
+              </CelText>
+              <CelText color={getColor(COLOR_KEYS.WHITE)} margin={"0 20 0 10"}>
+                For your security, it is strongly advised that you do NOT screenshot your two-factor authentication
+                code.
+              </CelText>
+            </View>
+          </View>
+        </InfoBox>
+
         <Card styles={{ alignItems: "center", marginTop: 25 }}>
```
Comment (CrnogoracIvan) [inline app/components/screens/TwoFactorSettings/TwoFactorSettings.js:104]: Move to styles.

**Review by CrnogoracIvan — CHANGES_REQUESTED** [review]
From comments.

File: app/components/screens/HODLViewCode/HODLViewCode.js [hunk 6]
```diff
@@ -68,9 +69,37 @@ class HODLViewCode extends Component {
     }
   };
 
+  function;
```
Comment (fjovakaric) [inline app/components/screens/HODLViewCode/HODLViewCode.js]: ?

**Review by fjovakaric — APPROVED** [review]
